buildkite/src/Command/ExecuteIntegrationTest.dhall

```diff
@@ -0,0 +1,47 @@
+let Prelude = ../External/Prelude.dhall
+
+let Command = ./Base.dhall
+let Docker = ./Docker/Type.dhall
+let Size = ./Size.dhall
+
+let Cmd = ../Lib/Cmds.dhall
+
+let deployEnv = "DOCKER_DEPLOY_ENV"
+
+in
+
+{ step = \(testName : Text) -> \(dependsOn : List Command.TaggedKey.Type) ->
+    Command.build
+      Command.Config::{
+        commands = [
+            Cmd.run (
+              "if [ ! -f ${deployEnv} ]; then " ++
```

Comment:
created https://github.com/MinaProtocol/mina/issues/7745 to address DRY'ing this code up a bit